securedrop/tests/test_integration.py

```diff
@@ -176,7 +178,10 @@ def test_submit_file(source_app, journalist_app, test_journo):
         decrypted_data = journalist_app.crypto_util.gpg.decrypt(resp.data)
         assert decrypted_data.ok
 
-        sio = BytesIO(decrypted_data.data)
+        if six.PY2:  # Python2
```

Comment:
This feels logically wrong. In the python2 path, we're using "write bytes" against a `StringIO` which feels like it could break under non-Ascii strings. I think we should use `six.moves.StringIO` and then an if/else to ensure that we're using the `six.string_type`.

This is test-only code, so if this passes CI, we might be ok. I just worry that as we move more of the code over to using `six` that this may break.

Reply:
I am not modifying any test code here, I am keeping what ever is there, we can modify the test in future, but in a different PR.

Reply:
I think what we want here is `six.BytesIO` - `StringIO.StringIO` on Python 2, `io.BytesIO` on Python 3

Reply:
(though this did pass CI)